src/components/TileCatalogNew/TilePagination/TilePagination.jsx

```diff
@@ -86,13 +86,105 @@ const TilePagination = ({ page, numPages, onChange, i18n }) => {
       {pageNumber}
     </button>
   );
+  const getPageSelect = (pageNumber, accumulator) => (
+    <li className="bx--pagination-nav__list-item">
+      <div className="bx--pagination-nav__select">
+        <select
+          className="bx--pagination-nav__page bx--pagination-nav__page--select"
+          data-page-select
+          aria-label="select page number"
```

Comment:
localization

Reply:
Done. 
And there is a new story under TileCatalogNew/With OverflowMenu in Pagination.